ark/phenotyping/som_utils.py

```diff
@@ -9,14 +9,14 @@
 from skimage.io import imread
 
 import ark.settings as settings
+from ark.analysis import visualize
 from ark.utils import io_utils
 from ark.utils import load_utils
 from ark.utils import misc_utils
 
 
-def compute_cluster_avg(fovs, channels, base_dir,
-                        cluster_dir='pixel_mat_clustered',
-                        cluster_avg_name='pixel_cluster_avg.feather'):
+def compute_cluster_avg(fovs, channels, base_dir, cluster_col,
+                        cluster_dir='pixel_mat_clustered'):
     """Averages channel values across all fovs in pixel_mat_clustered
```

Comment:
A bit more description in docstring: The averages being computed are within each cluster